src/databased/runtime.rs

```diff
@@ -194,13 +219,22 @@ impl Runtime {
                     swap_id,
                     service_id: ServiceId::Wallet,
                 }) {
-                    debug!("{} | Did not delete checkpoint entry: {}", swap_id, err);
+                    debug!(
```

Comment:
Wondering if `debug` is correct or if it should be `warn` level (or `error`)?

If we switch to `warn` or `error` we can add colors to the output with `swap_id.swap_id()`